.ruby-version

```diff
@@ -1 +0,0 @@
-ruby-2.2
```

Comment:
can we leave this in for now. Not quite sure what to do going forward with this. I introduced it to try stabilise the old stuff, might get rid of it soon

Reply:
Ok -- but 2.2 is a weird choice since it is EOL

Reply:
Agreed. I am sorry. It is something reasonably high on my list. It's also one of the few things I can actually help with.